examples/eSealing/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "e_sealing"
+version = "0.1.0"
+authors = ["Concordium <developers@concordium.com>"]
+edition = "2018"
+license = "MPL-2.0"
+
+[features]
+default = ["std"]
+std = ["concordium-std/std"]
```

Comment:
these feature flags only makes sense if the contract supports no_std, which is seems not to do.
So maybe remove them and allow the default features of `concordium-std`

Reply:
I added `#![cfg_attr(not(feature = "std"), no_std)]` into the `lib.rs` file.